charts/testkube-api/templates/deployment.yaml

```diff
@@ -292,73 +142,6 @@ spec:
               value:  "{{ .Values.cliIngress.oauth.provider }}"
             - name: TESTKUBE_OAUTH_SCOPES
               value:  "{{ .Values.cliIngress.oauth.scopes }}"
-            {{- if .Values.cloud.key }}
```

Comment:
These `OAUTH_` variables are not being used in the agent binary, @vsukhin do you think can these be removed all together?

Reply:
https://github.com/kubeshop/helm-charts/pull/868/files/650c28b26060d609b15e614b1985b01853a1eca5#diff-b3d5a53daf2423ea63a61fbf1d77611d46673954cfe7cfcf6e8dd059204708e3L287-L293

Reply:
sounds like, they were not removed together with dashboard ssuport in cli